### PR TITLE
Building a FirrtlInterpreterRepl tutorial.

### PR DESCRIPTION
**Pull request by chick**
Don't merge this yet.  Please comment on style, content

### CODE REVIEW

File: Debugging-with-the-Interpreter-REPL-1.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+    x := io.value1
+    y := io.value2
+  }
+
+  io.outputGCD := x
+  io.outputValid := y === 0.U
+}
+```
+Using your favorite code editor, let's break it as follows.
+Currently the declarations of the ```x``` and ```y``` are using Chisels width inference to determine their number of bits.
```
Comment (ucbjrl) [inline Debugging-with-the-Interpreter-REPL-1.md]: Chisels -> Chisel's

File: Debugging-with-the-Interpreter-REPL-2.md [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+firrtl>> poke io_loadingValues 0
+firrtl>> waitfor io_outputValid 1 100
+io_outputValid == value 1 in 3 cycles
+firrtl>> peek io_outpugGCD
+peek io_outputGCD  5
+```
+We poke 10 and 15 and wait for output valid, and when it is after 3 cycles the correct value 5 is on the output.
+
+### What's next
+Check out [Debugging with the Interpreter-REPL VCD Page 3](Debugging-with-the-Interpreter-REPL-3)
```
Comment (ucbjrl) [inline Debugging-with-the-Interpreter-REPL-2.md]: Should this missing file (Debugging-with-the-Interpreter-REPL-3.md)be part of this PR?

File: Home.md [hunk 3]
```diff
@@ -27,4 +27,5 @@ If you're developing a Chisel library, see [Developers](Developers).
 * [Scala Things You Should Know](Scala-Things-You-Should-Know)
 * [Scala-land vs. Chisel-land](Scala-land-vs.-Chisel-land)
 * [CS250 a Chisel+Scala Primer](CS250-Chisel+Scala-Primer)
+* [Debugging with the Interpreter-REPL](Debugging-with-the-Interpreter-REPL-1)
```
Comment (ucbjrl) [inline Home.md]: Is the fact that we only show the first of the two (three?) pages intentional?

Comment (chick) [reply Home.md]: I was planning on finishing the 3rd as part of this PR if you thought these two pages look ok.

**Comment by chick** [comment]
Bigger question is whether you think this belongs in chisel3 or firrtl-interpreter or even chisel-testers wiki?

**Comment by ucbjrl** [comment]
We could discuss this, but my vote is for the chisel3 wiki. As things progress, I suspect we will eventually want to move it to the firrtl-interpreter wiki (with appropriate links from chisel3), but it would be awfully lonely there at the moment.

File: Home.md (hunk 3 above)
Comment (ucbjrl) [reply Home.md]: I think they look great.

**Comment by chick** [comment]
Agreed on putting it in chisel3 wiki

**Pull request merged**